asset-manager/README.md

```diff
@@ -181,72 +112,175 @@ scripts/start.sh
+   - **Task Prompt**: You are a Spring Boot developer assistant, follow the Spring Boot Actuator documentation to add basic health endpoints for Azure Container Apps deployment.
+1. Click the **Add References** button to add the Spring Boot Actuator official documentation as references.
+
+   ![Health endpoint task](doc-media/health-endpoint-task.png)
+1. In the popped-up quick-pick window, select **External links**. Then paste the following link: `https://docs.spring.io/spring-boot/reference/actuator/endpoints.html`. Click **Save** to create the task.
+1. Click the **Run** button to trigger the custom task.
+1. Follow the same steps as the predefined task to review and apply the changes.
+1. Review the proposed code changes and click **Keep** to apply them.
+
+### Containerize Applications
+
+Now that you have successfully migrated your Java application to use Azure services, the next step is to prepare it for cloud deployment by containerizing both the web and worker modules. In this section, you will use **Containerization Tasks** to containerize your migrated applications.
+> Note: If you have any problems with the previous migration step, you can start from the [workshop/expected](https://github.com/Azure-Samples/java-migration-copilot-samples/tree/workshop/expected/asset-manager) branch for the deployment operation.
+
+1. Open the sidebar of `GITHUB COPILOT APP MODERNIZATION`. In **Tasks** view, click the **Run Task** button of **Java** -> **Containerization Tasks** -> **Containerize Application**.
+  
+    ![Run Containerize Application task](doc-media/containerization-run-task.png)
+
+1. A predefined prompt will be populated in the Copilot Chat panel with Agent Mode. Copilot Agent will start to analyze the workspace and to create a **containerization-plan.copiotmd** with the containerization plan.
+
+    ![Containerization prompt and plan](doc-media/containerization-plan.png)
+1. View the plan and collaborate with Copilot Agent as it follows the **Execution Steps** in the plan by clicking **Continue**/**Allow** in pop-up chat notifications to run commands. Some of the execution steps leverage agentic tools of **Container Assist**.
+
+    <!-- ![Containerization execution steps](doc-media/containerization-execution-steps.png) -->
+1. Copilot Agent will help generate Dockerfile, build Docker images and fix build errors if there are any. Click **Keep** to apply the generated code.
+
+### Deploy to Azure
+
+At this point, you have successfully migrated the sample Java application `asset-manager` to Azure Database for PostgreSQL (Spring), Azure Blob Storage, and Azure Service Bus, and exposed health endpoints via Spring Boot Actuator.
+> Note: If you have any problems with the previous migration step, you can start from the [workshop/expected](https://github.com/Azure-Samples/java-migration-copilot-samples/tree/workshop/expected/asset-manager) branch for the deployment operation.
```

Comment:
Note: If you encounter any issues with the previous migration step, you can directly proceed with the deployment step using the workshop/expected branch.